docs/README.md

```diff
@@ -15,11 +15,38 @@ We usually only install one application per VM; the current exception is the san
 
 In a few instances, we are using PUL cloud VMs for more experimental apps that are not production level but which we do not want restricted to campus IPs. This currently only applies to GeoTaste and SimRisk. We're using Google Cloud servers, but AWS is available. PUL typically uses these for short-term experimentation and testing of new applications before configuring and automating deployment on internal VMS.
 
-### Access to PUL VMs
+## Access to PUL VMs (How to Add a New Person to PUL VMs)
 
-CDH RSE team members have access PUL-managed servers using ssh and the `pulsys` (PUL Systems) account.
-Access is based on ssh keys associated with team member GitHub accounts; this is managed
-in [princeton_ansible variables](https://github.com/pulibrary/princeton_ansible/blob/main/group_vars/all/vars.yml#L115), and we can make request changes via pull requests. If you need to update ssh keys on a particular server (e.g., a newly provisioned VM), use the
-[Update pulsys user keys](https://ansible-tower.princeton.edu/#/templates/job_template/17/details) template in [Ansible Tower](https://ansible-tower.princeton.edu). When you launch the job, use the `limit` input to specify the hostnames for the servers you want to update so that it does not have to run on all PUL VMs.
+### Step 1: Add GitHub account to [princeton_ansible variables](https://github.com/pulibrary/princeton_ansible/blob/main/group_vars/all/vars.yml#L115)
 
-The `pulsys` account has `sudo all` permission. We also have a non-privileged deploy account `conan` (the deployer) which is used for actually running applications.
+CDH RSE team members access PUL-managed servers using SSH via the `pulsys` (PUL Systems) account.
+
+Access requires SSH keys linked to the team member's GitHub account. These are managed in [princeton_ansible variables](https://github.com/pulibrary/princeton_ansible/blob/main/group_vars/all/vars.yml#L115). Submit a pull request to add an entry under 'cdh_github_keys' (example format: [https://github.com/example_user.keys](https://github.com/tanhaow.keys)).
+
+To update SSH keys on a specific server (such as a new VM), use the [Update pulsys user keys](https://ansible-tower.princeton.edu/#/templates/job_template/17/details) template in [Ansible Tower](https://ansible-tower.princeton.edu/).
+
+### Step 2: Update SSH key
+
+Check [https://github.com/example_user.keys](https://github.com/tanhaow.keys). If the page is empty, you need to add an SSH key to your GitHub account following: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
+
+After adding the SSH key, refresh the page to verify it appears.
+
+### Step 3: Princeton Ansible Tower
+
+1. Navigate to [Princeton Ansible Tower](https://ansible-tower.princeton.edu/#/home)
+2. Go to 'templates' on the side bar
+3. Find the Job template 'Update pulsys user keys'
+4. Click 'launch'
```

Comment:
I initially thought it was an error that you had 'click launch' twice, but looked again and saw that you're right! Maybe we could add some brief context to clarify what it's doing.
```suggestion
4. Click 'launch' to open the dialog box to configure how to run the template
```